controllers/authrealm/dex.go

```diff
@@ -415,16 +430,24 @@ func (r *AuthRealmReconciler) createDexConnectors(authRealm *identitatemv1alpha1
 						Namespace: authRealm.Namespace,
 					},
 					UserSearch: dexoperatorv1alpha1.UserSearchSpec{
-						BaseDN:    authRealm.Spec.LDAPExtraConfigs[idp.Name].BaseDN,
-						Filter:    authRealm.Spec.LDAPExtraConfigs[idp.Name].Filter,
-						Username:  idp.LDAP.Attributes.PreferredUsername[0],
+						BaseDN:    url.BaseDN,
```

Comment:
Let's update the unit tests to be sure we're confirming these values are being mapped from the URL.

Reply:
updated